.github/workflows/build-macos.yml

```
@@ -83,10 +83,10 @@ jobs:
         # of these builds take roughly the same time
         include:
           - { 
-            name: "x86_64 (CPython 3.6 3.10 and above)",
+            name: "x86_64 (CPython 3.10 and above)",
             macarch: x86_64,
-            # pattern matches 6 or any 2 digit number
-            pyversions: "cp3{6,[1-9][0-9]}-*"
+            # pattern matches any 2 digit number
+            pyversions: "cp3{10,[1-9][0-9]}-*"
```

```suggestion
            pyversions: "cp3[1-9][0-9]-*"
```
The explicit 10 here is not needed, the pattern covers for this